subiquity/server/controllers/tests/test_userdata.py

```diff
@@ -39,7 +51,9 @@ def test_load_autoinstall_data(self):
             "ssh_import_id: 'wrong' is not of type 'array'"
         )
         invalid_schema = {"ssh_import_id": "wrong"}
-        validate = self.controller.app.base_model.validate_cloudconfig_schema
+        validate = (
+            self.controller.app.base_model.validate_cloudconfig_schema
+        ) = mock.Mock()
```

Comment:
The jobs of automatic code formatting :man_shrugging:

Reply:
I know right :upside_down_face: